packages/mutation-server-protocol/src/schema.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * Use named imports to improve tree-shaking capabilities.
+ */
+import {
+  array,
+  number,
+  object,
+  string,
+  enum as enum_,
+  type z,
+  boolean,
+} from 'zod';
+
+export const ConfigureParams = object({
+  /**
+   * The (relative or absolute) path to mutation testing framework's config file to load.
+   */
+  configFilePath: string().optional(),
+});
+export type ConfigureParams = z.infer<typeof ConfigureParams>;
+
+export const ConfigureResult = object({
+  /**
+   * The mutation testing server protocol major version that the client supports (major)
```

Comment:
I think this should read 'server' instead of 'client'. Or to be rephrased as 'the major version the client should support'

Reply:
Yes we discussed this previously and determined that only the client can practically deal with multiple versions so the server should simply return the protocol version they have implemented

Reply:
Hmmm. I can make it a constant value in the schema (`"1"`). That way, you cannot provide another value until we release version 2; at this point, we make it `"1"` or `"2"`.

What do you think?